Kernel/Clause.hpp

```diff
@@ -398,7 +412,8 @@ class Clause
   static bool _auxInUse;
 #endif
 
-//#endif
+  Literal* _answerLiteral;
```

Comment:
Could this be an `unsigned` (possibly with fewer bits) indicating the position of the literal in the clause?

Reply:
As discussed offline, we will keep it as a pointer, since not all clauses have an answer literal.